Features/seperate wind on offshore

MarlonSchlemminger opened the pull request.

MarlonSchlemminger committed:
seperate wind on- and offshore

MarlonSchlemminger committed:
seperate wind on- and offshore

MarlonSchlemminger committed:
foreign offshore feedin

MarlonSchlemminger committed:
change calculation offshore for neighbours

MarlonSchlemminger committed:
Merge branch 'dev' into features/seperate-wind-on-offshore

wolfbunke commented:
@MarlonSchlemminger Looks good thanks for the changes! I have only one question in   [ dataprocessing/sql_snippets/ego_dp_powerflow_create_pp_mview.sql](https://github.com/openego/data_processing/pull/233/commits/7426c69aad9778def1283821282a699057322285#diff-057bc5658220f58f3bce2072081b66c7L205)  you changed the MView of the RE plants of scenario ego100. Was there any reason or error for changing it?

MarlonSchlemminger commented:
The line I deleted was code I merged into dev last time on accident. I already deleted it in the dev branch a while ago, see this commit:
https://github.com/openego/data_processing/commit/7faccf587eabe66e7fe31c674079c55a2a2f3668
Do you agree? I was insecure about this but got errors if I left the line in. 
The same problem applies to these lines, I also asked @IlkaCu about this. My last merge was a bit confusing and I still don't know how it happened, sorry for that.: https://github.com/openego/data_processing/blob/dev/dataprocessing/sql_snippets/ego_dp_powerflow_assignment_generator.sql#L283-L348
